core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/ngbd-modal-workflow-executions/ngbd-modal-workflow-executions.component.html

```diff
@@ -72,12 +72,14 @@ <h4 class="modal-title">Execution results of Workflow: {{workflowName}}</h4>
   </ng-template>
   <ng-template #actionDelete>
     <i
+      (nzOnConfirm)="onGroupDelete()"
+      nz-popconfirm
+      nzPopconfirmTitle="Are you sure delete this execution group?"
```

Comment:
Minor comment, is this sentence correct in terms of grammar?

Reply:
Changed to "Confirm to delete this execution group."

Reply:
please update the PR description (the figure) as well.

Reply:
Updated.